examples/C++/RTPSTest_persistent/TestWriterPersistent.h

```diff
@@ -40,7 +40,7 @@ class TestWriterPersistent {
 	public:
 		MyListener():n_matched(0){};
 		~MyListener(){};
-		void onWriterMatched(eprosima::fastrtps::rtps::RTPSWriter*, eprosima::fastrtps::rtps::MatchingInfo& info)
+		void on_writer_matched(eprosima::fastrtps::rtps::RTPSWriter*, eprosima::fastrtps::rtps::MatchingInfo& info)
```

Comment:
I see this change now (similar in other examples below) and I am not sure, as we said we shouldn't need to change existing C++ examples.

Reply:
Reverted change. Added `override` in some of the tests to raise a compilation error in the future.